Gemfile.lock

```diff
@@ -310,8 +309,7 @@ GEM
     netrc (0.11.0)
     newrelic_rpm (9.16.0)
     nio4r (2.7.3)
-    nokogiri (1.18.10)
-      mini_portile2 (~> 2.8.2)
+    nokogiri (1.18.10-arm64-darwin)
```

Comment:
Something funky happened when adding the gem, it shouldn't be locking to a platform 🤔